plutus-core/plutus-ir/test/PlutusIR/Purity/ifThenElse

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
Where's the golden file for this?

Reply:
[plutus-core/plutus-ir/test/PlutusIR/Purity/ifThenElse.golden](https://github.com/IntersectMBO/plutus/pull/6974/files#diff-fffcfed6d95a46514c41ab031b645d30844b4d5b71a6f62c24871e04ad342a17)